PINFuture/Classes/Categories/NSURLSession+PINFuture.h

```diff
@@ -9,8 +9,10 @@
 #import <Foundation/Foundation.h>
 #import <Foundation/NSURLSession.h>
 
-#import "PINNSURLSessionDataTaskAndResult.h"
-#import "PINFuture.h"
+#import <PINFuture/PINNSURLSessionDataTaskAndResult.h>
+#import <PINFuture/PINFuture.h>
```

Comment:
i think it is uncommon to include umbrella header in its header file?

seems <PINFuture/PINFuture.h> includes NSURLSession+PINFuture.h